hack/lib/cleanup.sh

```diff
@@ -95,7 +95,7 @@ function os::cleanup::tmpdir() {
     fi
 
 		# delete any sub directory underneath BASETMPDIR
-    for directory in $( find "${BASETMPDIR}" -d 2 ); do
+    for directory in $( find "${BASETMPDIR}" -depth 2 ); do
```

Comment:
Will drop this change. There is still some error messages, might be Mac related.